.github/workflows/main.yml

```diff
@@ -361,6 +359,18 @@ jobs:
       platform: linux-x64
       bootjdk-platform: linux-x64
       runs-on: ubuntu-22.04
+      debug-suffix: -debug
+
+  test-linux-x64-static:
+    name: linux-x64-static
+    needs:
+      - build-linux-x64-static
```

Comment:
Just thinking out loud here, what if you also added ` build-linux-x64` to the `needs` list. I guess that would make sure the test bundle is built before trying to execute this. Then the trick will just be to get both the test bundle and the static-jdk bundle downloaded...

Reply:
> Just thinking out loud here, what if you also added ` build-linux-x64` to the `needs` list. I guess that would make sure the test bundle is built before trying to execute this. Then the trick will just be to get both the test bundle and the static-jdk bundle downloaded...

That's an interesting idea. I'll give it a try. Thanks!

Reply:
@magicus I looked into this today. Adding `build-linux-x64` to the `needs` list for `test-linux-x64-static` works with additonal changes in .github/actions/get-bundles/action.yml to download the static bundle separately. I added a `download-static-bundles` step for the static bundle. 

With this, I was able to only build the `static-jdk-bundles` without `product-bundles test-bundles` in `build-linux-x64-static` job, as your other suggestion.